swap_meet/vendor.py

```diff
@@ -1,2 +1,106 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory= None): 
+        self.inventory = inventory if inventory is not None else []
+    
+# Methods 
+
+    def add(self, item):
+        ''' adds item to inventory '''
+        self.inventory.append(item)
+        return item 
+
+
+    def remove(self, other_item):
+        ''' removes duplicate item if true '''
+        if other_item in self.inventory:
+            self.inventory.remove(other_item)
+            return other_item
+        else:
+            return False
```

Comment:
Let's swap these conditionals to follow common guard clause styling

```suggestion
        if other_item not in self.inventory:
            return False
        self.inventory.remove(other_item)
        return other_item
```

feel free to add in an `else` to remain consistent with the rest of your code.